pylintrc

```diff
@@ -1,5 +1,6 @@
 [MASTER]
-ignore-patterns=test_*,azure_devops_build_*
+ignore=tests,vendored_sdks,privates
+ignore-patterns=test.*,azure_devops_build.*
```

Comment:
The original config value `test_*` is wrong ❌. According to http://pylint.pycqa.org/en/latest/technical_reference/features.html

> Files or directories matching the **regex patterns** are skipped. The regex matches against base names, not paths.

It should be regex, instead of wildcard. 

`test_*` is used to match `test________...`. It works because `_*` doesn't match anything at all and only `test` is used to do the regex match.